Core.DomainModel/Core.DomainModel.csproj

```diff
@@ -243,6 +243,7 @@
     <Compile Include="Result\OperationError.cs" />
     <Compile Include="Result\OperationFailure.cs" />
     <Compile Include="Result\Result.cs" />
+    <Compile Include="Shared\YearMonthIntervalOption.cs" />
```

Comment:
Det er et ret specifikt udfaldsrum som er knyttet til GDPR DP domænet så den skal ikke være i shared. Den er for specifik. Ind under GDPR med den